src/batch_normalization.js

```diff
@@ -20,7 +20,7 @@ export function batchNormalization(input, mean, variance, {axis=1, scale, bias,
   // The output tensor has the same shape as the input tensor.
   let output = new Tensor(input.shape);
   const shape = new Array(input.rank).fill(1);
-  shape[axis] = -1;
+  shape[axis] = null;
```

Comment:
Ah yes, we still have this special case with Reshape that was not changed by Ningxin's PR. Do you think it worth opening an issue to simplify the implementation here too? Generally I'd expect that the calling frameworks know best how to apply any special policies and can already resolve these cases before it reaches the WebNN API. (this comment is not blocking)

Reply:
> to simplify the implementation

Sorry, I don't catch your point, could you please explain how to simplify the implementation?

And I submitted a pr https://github.com/webmachinelearning/webnn/pull/367 to Spec, please also have a review, thanks.

Reply:
👍 Commented on the other PR.